[2025-10-15T11:17:42.457+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: translation_pipeline.notify_translation_completion manual__2025-10-15T11:11:36.868017+00:00 [queued]>
[2025-10-15T11:17:42.463+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: translation_pipeline.notify_translation_completion manual__2025-10-15T11:11:36.868017+00:00 [queued]>
[2025-10-15T11:17:42.463+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-10-15T11:17:42.471+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): notify_translation_completion> on 2025-10-15 11:11:36.868017+00:00
[2025-10-15T11:17:42.476+0000] {standard_task_runner.py:60} INFO - Started process 296 to run task
[2025-10-15T11:17:42.478+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'translation_pipeline', 'notify_translation_completion', 'manual__2025-10-15T11:11:36.868017+00:00', '--job-id', '1405', '--raw', '--subdir', 'DAGS_FOLDER/translation_pipeline.py', '--cfg-path', '/tmp/tmpyog9ks1n']
[2025-10-15T11:17:42.479+0000] {standard_task_runner.py:88} INFO - Job 1405: Subtask notify_translation_completion
[2025-10-15T11:17:42.523+0000] {task_command.py:423} INFO - Running <TaskInstance: translation_pipeline.notify_translation_completion manual__2025-10-15T11:11:36.868017+00:00 [running]> on host 289cda1aef62
[2025-10-15T11:17:42.591+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='pdf-converter' AIRFLOW_CTX_DAG_ID='translation_pipeline' AIRFLOW_CTX_TASK_ID='notify_translation_completion' AIRFLOW_CTX_EXECUTION_DATE='2025-10-15T11:11:36.868017+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-15T11:11:36.868017+00:00'
[2025-10-15T11:17:42.603+0000] {translation_pipeline.py:741} ERROR - ❌ Ошибка отправки уведомления: Нет данных о результате перевода в XCom save_translation_result
[2025-10-15T11:17:42.603+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-15T11:17:42.614+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=translation_pipeline, task_id=notify_translation_completion, execution_date=20251015T111136, start_date=20251015T111742, end_date=20251015T111742
[2025-10-15T11:17:42.649+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-15T11:17:42.666+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
